package.json

```diff
@@ -6,18 +6,25 @@
   "scripts": {
     "integrationTest": "node ./node_modules/mocha/bin/mocha test/integration/main.test.js --reporter spec",
     "test": "node ./node_modules/mocha/bin/mocha test/unit/*.test.js --reporter spec",
-    "start": "node index.js"
+    "start": "node index.js",
+    "migrate:make": "./node_modules/.bin/knex migrate:make",
```

Comment:
You can write `"migrate:make": "knex migrate:make",` and npm will find knex inside `./node_modules/.bin/` himself.